views/address.tmpl

```diff
@@ -56,24 +56,38 @@
                         </tr>
                         {{end}}
                     {{else}}
-                    <tr>
-                        <td class="pr-2 lh1rem vam text-right xs-w91">UNCONFIRMED</td>
-                        <td class="fs28 mono nowrap fs16-decimal d-flex align-items-center justify-content-end">
-                            {{.NumUnconfirmed}}
-                        </td>
+                    <tr  class="h2rem">
+                        <td class="pr-2 lh1rem vam text-right xs-w91">TOTAL UNSPENT</td>
+                        {{if eq .KnownFundingTxns .MaxLimit}}
+                        <td class="fs28 mono nowrap d-flex align-items-center justify-content-end">unavailable</td>
+                        {{else}}
+                        <td class="fs28 mono nowrap fs16-decimal d-flex align-items-center justify-content-end">{{.Unspent}}</td>
+                        {{end}}
                     </tr>
                     <tr>
                         <td class="text-right pr-2">RECEIVED</td>
+                        {{if eq .KnownFundingTxns .MaxLimit}}
+                        <td class="mono nowrap text-right">unavailable</td>
+                        {{else}}
                         <td class="mono nowrap text-right">{{.TotalReceived}}</td>
+                        {{end}}
                     </tr>
                     <tr>
-                        <td class="text-right pr-2">SENT</td>
+                        <td class="text-right pr-2">SPENT</td>
+                        {{if eq .KnownFundingTxns .MaxLimit}}
+                        <td class="mono nowrap text-right">unavailable</td>
+                        {{else}}
                         <td class="mono nowrap text-right">{{.TotalSent}}</td>
+                        {{end}}
                     </tr>
-                    <tr>
-                        <td class="text-right pr-2">UNSPENT</td>
-                        <td class="mono nowrap text-right">{{.Unspent}}</td>
-                    </tr>
+                    {{if ne .NumUnconfirmed 0}}
+                        <tr>
+                            <td class="text-right pr-2">UNCONFIRMED</td>
```

Comment:
This is only in the lite mode loop but should apply to both modes. Only needs to be coded once.